### PR TITLE
[WIP] Torris Jungle Base (NSB Forbearance)

### PR DESCRIPTION
**Pull request by Rykka-Stormheart**
Welcome to the NSB Forbearance on Torris!

An extremely warm, humid jungle planet in [random system] with a large, NT Research base in one of the larger jungles!

Your base is above the treetops, with a ground 'vehicle' area and small complex at the very base. Hungry panthers and other jungle critters wander around, so venture beyond the fence at your own peril!

View current doodles here:
Verkister V1:

Verkister V2:

Scree:

This is a DRAFT and is intended more as a show of progress and to allow collaborators to start working on the project!

List of Objectives:
### Pre-Alpha Stage
- [x] Lay out Medical
- [x] Lay out Bridge
- [x] Lay out Science Outpost (Toxins/Xenobio/Xenoarch)
- [ ] Lay out Engineering
- [ ] Lay out Cargo
- [ ] Lay out Research
- [ ] Lay out Exploration
- [ ] Lay out Security
- [ ] Lay out Botany/Xenoflora/Xenobotany 
- [ ] Lay out Bar/Kitchen
- [ ] Lay out Dorms
- [ ] Lay out Tram + Arrivals
- [ ] Lay out Chapel/Library/Misc Civ Areas (Reading Room, Auditorium, Holodeck)
- [ ] Lay out Cafe
- [ ] Lay out Engine/Solars
- [ ] Lay out AI Bunker
- [ ] Finish Base Station Layout (Maint/Halls/Misc areas)
### Alpha Stage
- [ ] Flesh Out Medical
- [ ] Flesh Out Bridge
- [ ] Flesh Out Engineering
- [ ] Flesh Out Cargo
- [ ] Flesh Out Research
- [ ] Flesh Out Exploration
- [ ] Flesh Out Security
- [ ] Flesh Out Science Outpost (Toxins/Xenobio/Xenoarch)
- [ ] Flesh Out Botany/Xenoflora/Xenobotany 
- [ ] Flesh Out Bar/Kitchen
- [ ] Flesh Out Dorms
- [ ] Flesh Out Tram + Arrivals
- [ ] Flesh Out Chapel/Library/Misc Civ Areas (Reading Room, Auditorium, Holodeck)
- [ ] Flesh Out Cafe
- [ ] Flesh Out Engine/Solars
- [ ] Flesh Out AI Bunker
- [ ] Flesh Out Maintenance/Hallways
- [ ] Add Wilderness POIs
- [ ] First pass on small details/proof-check
- [ ] Gather pre-beta-test feedback
- [ ] Add code framework for day/night sounds, all mobs, unique map features
### Beta Stage
- [ ] Playtest 1
- [ ] Review feedback, iterate
- [ ] Playtest 2
- [ ] Review feedback, iterate
- [ ] Playtest 3
- [ ] Review feedback, iterate
### Release Stage
- [ ] Celebrate (cry?)

### CODE REVIEW

**Comment by KillianKirilenko** [comment]
Interesting idea with the cable car 'tram'. I had a similar idea for an old temperate colony concept, except it was the way you'd reach the 'outpost' (backup power w/ solars/wind and mining) rather than being the main way in and out (that was handled by trains, which also served cargo).

~~Can't wait to see how much damage tesh will take from accidentally yeeting themselves over walkway railings down to the jungle floor.~~

**Comment by KasparoVy** [comment]
Welcome to the Jungle :0

**Comment by TheFurryFeline** [comment]
I see auto changelog workflow was active in your fork. You'll want to revert that commit and disable any Actions not specifically for checking prs for errors.

**Comment by Rykka-Stormheart** [comment]
> I see auto changelog workflow was active in your fork. You'll want to revert that commit and disable any Actions not specifically for checking prs for errors.

Er, how do I do that?

**Comment by TheFurryFeline** [comment]
> > I see auto changelog workflow was active in your fork. You'll want to revert that commit and disable any Actions not specifically for checking prs for errors.
> 
> Er, how do I do that?

Disabling via your fork, just pop onto it and go to the Actions tab and find the autochangelog.yml and the other that isn't ci.yml that's specifically for running checks on the codebase. Am not at home to make screenshots, apologies

**Comment by TheFurryFeline** [comment]
> > > I see auto changelog workflow was active in your fork. You'll want to revert that commit and disable any Actions not specifically for checking prs for errors.
> > 
> > 
> > Er, how do I do that?
> 
> Disabling via your fork, just pop onto it and go to the Actions tab and find the autochangelog.yml and the other that isn't ci.yml that's specifically for running checks on the codebase. Am not at home to make screenshots, apologies

Sorry for being late to this, but got that screenshot now on how to disable your undesired workflows 
image Go to your fork, click the actions tab, and below the "create status badge" button will be "disable workflow", which had already been done in this screenshot. 

As for reverting a commit that fiddled with the autochangelog.yml, you'll have to find that one in a third party app, I know of no way of doing it via browser GitHub. Actually, lemme see if I can find it and revert that one for ya.... I remember I still have access to the repo... ... ... Or at least I used to, heck. Ah, well, I'll just PR it to this branch.

File: code/game/objects/structures/flora/flora_vr.dm [hunk 1]
```diff
@@ -73,29 +73,115 @@
...
+	. = ..()
+
+/datum/category_item/catalogue/flora/torris/nyrian
+	name = "Torris Flora - Nyrian Bloom"
+	desc = "*error* _datanotfound: please try again." //WIP
+	value = CATALOGUER_REWARD_EASY
+
+/obj/structure/flora/torris/nyrian
+	name = "pink flower"
+	desc = "An odd looking flower bloom that has a slight haze of pollen surrounding it. It smells very strange ."
```
Comment (TheFurryFeline) [inline code/game/objects/structures/flora/flora_vr.dm:140]:
```suggestion
	desc = "An odd looking flower bloom that has a slight haze of pollen surrounding it. It smells very strange."
```

File: code/game/objects/structures/flora/flora_vr.dm [hunk 2]
```diff
@@ -73,29 +73,115 @@
...
+	. = ..()
+
+/datum/category_item/catalogue/flora/torris/rafflesia
+	name = "Torris Flora - Rafflesia"
+	desc = "*error* _datanotfound: please try again." //WIP
+	value = CATALOGUER_REWARD_EASY
+
+/obj/structure/flora/torris/rafflesia
+	name = "strange flower"
+	desc = "A huge bright red flower that has an absolutly horrid smell when you draw near."
```
Comment (TheFurryFeline) [inline code/game/objects/structures/flora/flora_vr.dm:155]:
```suggestion
	desc = "A huge bright red flower that has an absolutely horrid smell when you draw near."
```

**Review by TheFurryFeline — COMMENTED** [review]
Minor fixes.

File: code/game/turfs/simulated/outdoors/virgoturf.dm [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+///////////NSB Forbearance///////////
+///////////Jungle Flooring///////////
+
+/datum/category_item/catalogue/flora/torris/jungle_grass
+	name = "Torris Flora - Sweetgrass"
+	desc = "A naturaly thick grass that has adapted to the hostile climate of Torris. \
+	The grass is dense and covers the moist ground of the surface of Torris. The grass \
+	can't grow or survive longanywhere else as it requires the dense moisture in the \
+	atmosphere to survive. When cut it gives off a plesantly sweet scent."
```
Comment (TheFurryFeline) [inline code/game/turfs/simulated/outdoors/virgoturf.dm:17]:
```suggestion
	desc = "A naturally thick grass that has adapted to the hostile climate of Torris. \
	The grass is dense and covers the moist ground of the surface of Torris. The grass \
	can't grow or survive long anywhere else as it requires the dense moisture in the \
	atmosphere to survive. When cut it gives off a pleasantly sweet scent."
```

File: code/modules/planet/torris.dm [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+	light_modifier = 0.3
+	flight_failure_modifier = 10
+	effect_message = "<span class='warning'>Rain falls on you, drenching you in water.</span>"
+
+	var/next_lightning_strike = 0 // world.time when lightning will strike.
+	var/min_lightning_cooldown = 5 SECONDS
+	var/max_lightning_cooldown = 1 MINUTE
+	observed_message = "An intense storm pours down over the region, lashing the trees in the wind."
+	transition_messages = list(
+		"You feel intense winds hit you as the weather takes a turn for the worst.",
```
Comment (TheFurryFeline) [inline code/modules/planet/torris.dm:216]:
```suggestion
		"You feel intense winds hit you as the weather takes a turn for the worse.",
```

File: maps/jungle_base/junglebase_jobs.dm [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+	sorting_order = -2
+	assignable = FALSE
+	visible = FALSE
+
+/datum/job/talon_captain
+	title = "Talon Captain"
+	flag = TALCAP
+	department_flag = TALON
+	departments_managed = list(DEPARTMENT_TALON)
+	job_description = "The captain's job is to generate profit through trade or other means such as salvage or even privateering."
```
Comment (TheFurryFeline) [inline maps/jungle_base/junglebase_jobs.dm:16]: Wait... Privateering? I thought Talon was specifically forbidden to be pirate?

**Review by TheFurryFeline — COMMENTED** [review]
Couple more fixes and a comment on Talon Captain job desc

**Comment by Arokha** [comment]
I like it, hope you continue on it at some point.

**Pull request closed without merging**